### PR TITLE
Release 1.34.0.0

### PR DESCRIPTION
**Pull request by bezirg**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Changelog fragments have been written (if appropriate)
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, PNG optimization, etc. are updated
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [ ] Targeting master unless this is a cherry-pick backport
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

File: cabal.project [hunk 1]
```diff
@@ -99,8 +99,8 @@ allow-newer:
 --
 -- package cardano-constitution
 --   flags: +force-build
--- allow-newer: cardano-constitution:plutus-ledger-api
--- allow-newer: cardano-constitution:prettyprinter-configurable
+-- allow-newer: *:plutus-ledger-api
+-- allow-newer: *:prettyprinter-configurable
```
Comment (Unisay) [inline cabal.project:103]: Why is this change needed?

File: cardano-constitution/cardano-constitution.cabal [hunk 2]
```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-constitution
-version:            1.32.0.0
+version:            1.34.0.0
```
Comment (Unisay) [inline cardano-constitution/cardano-constitution.cabal:3]: Did we skip 1.33 last release (I wonder why) ?

**Review by Unisay — APPROVED** [review]
LGTM

File: cardano-constitution/cardano-constitution.cabal (hunk 2 above)
Comment (bezirg) [reply cardano-constitution/cardano-constitution.cabal:3]: It's not released on chap atm, so it is fine

Comment (bezirg) [reply cardano-constitution/cardano-constitution.cabal:3]: Also the prepare-release.sh by design will not update it, which is as expected
